usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -3505,9 +3505,6 @@ static CK_RV import_DH_key(STDLL_TokData_t * tokdata, SESSION * sess,
          * Builds the DER encoding (ansi_x962) SPKI.
          * (get the length first)
```

Comment:
Remove '(get the length first)' from the comment

Reply:
Done